rootfs/etc/haproxy/template/haproxy.tmpl

```diff
@@ -165,6 +167,9 @@ backend {{ $backend.Name }}
             {{- if ge $slot.BackendEndpoint.Weight 0 }} weight {{ $slot.BackendEndpoint.Weight }}{{ end }}
         {{- else }} 127.0.0.1:81 disabled{{ end }}
         {{- template "backend" map $cfg $backend }}
+        {{- if eq $sticky.AffinityType "cookie" }}
+            {{- if not $sticky.CookieSessionAffinity.Dynamic }} cookie {{ $slot.BackendServerName }}{{- end }}
```

Comment:
This last `{{ end }}` might be without `-`.

Reply:
Fixed.